File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/converter/TspToCupConverter.java
package com.tarbadev.witchcraft.converter;

public class TspToCupConverter implements IConverter {
  private static final Double TSP_IN_ONE_CUP = 48.0;

  @Override
  public Double convert(Double number) {
    return number / TSP_IN_ONE_CUP;
  }
}

File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/converter/TspToCupConverter.kt
package com.tarbadev.witchcraft.converter

class TspToCupConverter : IConverter {

    override fun convert(number: Double): Double {
        return number / TSP_IN_ONE_CUP
    }

    companion object {
        private const val TSP_IN_ONE_CUP = 48.0
    }
}
